cmd/gke-gcloud-auth-plugin/cred/cred.go

```diff
@@ -127,12 +95,18 @@ type Options struct {
 //    }
 // }
 func PrintCred(options *Options) error {
-	if options == nil {
```

Comment:
Is it worth it to still check for null and initialize an empty options if necessary?

Reply:
Seems safer to keep, will add back